cli/run.go

```diff
@@ -114,27 +114,44 @@ func run(c *cli.Context) error {
 	prGlobalFlag := c.GlobalBool(PRKey)
 	ciGlobalFlag := c.GlobalBool(CIKey)
 
-	inventoryBase64Data := c.String(InventoryBase64Key)
-	inventoryPath := c.String(InventoryKey)
+	workflowToRunID := c.String(WorkflowKey)
+	if workflowToRunID == "" && len(c.Args()) > 0 {
+		workflowToRunID = c.Args()[0]
+	}
 
 	bitriseConfigBase64Data := c.String(ConfigBase64Key)
-
 	bitriseConfigPath := c.String(ConfigKey)
 	deprecatedBitriseConfigPath := c.String(PathKey)
 	if bitriseConfigPath == "" && deprecatedBitriseConfigPath != "" {
 		log.Warn("'path' key is deprecated, use 'config' instead!")
 		bitriseConfigPath = deprecatedBitriseConfigPath
 	}
+
+	inventoryBase64Data := c.String(InventoryBase64Key)
```

Comment:
shouldn't this be the other way around? to init the `params` struct from json (if provided)?